enzy_htp/structure/structure.py

```diff
@@ -771,13 +778,22 @@ def add(self, target: List[Residue], # pylint: disable=function-redefined
 
     @dispatch
     def absolute_index(self, target:Residue, indexed:int=0) -> int:
```

Comment:
What is this dispatching against?

Reply:
Nothing. I honestly don't remember why I put it. I'll remove lol.